include/rocksdb/db.h

```diff
@@ -1140,6 +1140,12 @@ class DB {
   virtual Status GetCurrentWalFile(
       std::unique_ptr<LogFile>* current_log_file) = 0;
 
+  // Retrieves the creation time of the oldest file in the DB.
+  // This API only works if max_open_files = -1, if it is not then
+  // Status returned is Status::NotSupported()
+  // The file creation time is set using the env provided to the DB.
```

Comment:
Also mention the behavior with DB generated by older release, or uncompacted files generated by older releases.